src/test/resources/features/EndToEnd.feature

```diff
@@ -103,7 +103,7 @@ Feature: HOCS is able to move cases through the entire flow
     And I complete the dispatch stage
     Then the case should no longer be visible in the teamqueue
 
-  @EndToEnd @Critical
+  @EndToEnd @Critical @SmokeTest
```

Comment:
Just a small thing - `@SmokeTests` instead of `@SmokeTest`

Reply:
Change made